app/helpers/photos_helper.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module PhotosHelper
+  REJECTED_TAGS = ( ExifMetadata::REJECTED_TAGS + %i[dc dimensions] ).freeze
+
+  def metadata_table( photo )
```

Comment:
Can we call this `metadata_table_rows` since that's what it renders?

Reply:
Yeah, that makes sense 👍